charts/kube-state-metrics/templates/deployment.yaml

```diff
@@ -195,6 +195,9 @@ spec:
         resources:
 {{ toYaml .Values.resources | indent 10 }}
 {{- end }}
+{{- with .Values.containerSecurityContext }}
```

Comment:
This uses a different style as above (if vs. with). Ideally we wouldn't mix those.

Reply:
makes sense; updated with the if style